### PR TITLE
More fixes to Numpy so that the binary is accepted by the App Store

### PR DESCRIPTION
**Pull request by lerela**
See #409 for context

### CODE REVIEW

File: recipes/numpy/numpy-1.16.4.patch [hunk 1]
```diff
@@ -12,15 +13,72 @@ diff -Naur numpy/core/setup.py numpy-1.16.4/numpy/core/setup.py
...
+index 66c07c9e1..817b45501 100644
+--- a/numpy/linalg/setup.py
++++ b/numpy/linalg/setup.py
+@@ -38,9 +38,12 @@ def configuration(parent_package='', top_path=None):
                  return []
              return [all_sources[0]]
  
 +    def get_lapack_lite_sources_ios(ext, build_dir):
-+        return []
++        return [all_sources[0]]
```
Comment (AndreMiras) [inline recipes/numpy/numpy-1.16.4.patch:81]: isn't this ending up behaving the same way as originally before being patched at all?
https://github.com/numpy/numpy/blob/v1.16.4/numpy/linalg/setup.py#L39

**Comment by lerela** [comment]
It seems to me that Numpy **is** referencing private Apple symbols as the file [umath_linalg.c.src](https://github.com/numpy/numpy/blob/v1.16.4/numpy/linalg/umath_linalg.c.src) wraps BLAS and LAPACK routines. Since `lapack_info` is not `None` in `numpy/linalg/setup.py`, Numpy's Lapack emulation is _not_ compiled, yet `umath_linalg.c.src` expects those symbols. 
They are found in some Apple framework (most likely Accelerate) during linking but as they are not public, Apple will reject the application later on.

An explanation could be that these symbols are only private on iOS, which would explain why Numpy is working fine on MacOS. Thanks for the suggestion, I will report these findings to Numpy.

File: recipes/numpy/numpy-1.16.4.patch (hunk 1 above)
Comment (lerela) [reply recipes/numpy/numpy-1.16.4.patch]: Correct, fixed

**Comment by AndreMiras** [comment]
OK that makes even more sense now thank you.
Thanks for editing/squashing the commit with detailed explanation. I'm ready for a merge once we have solved the conflicts.
We recently moved the source code (including `recipes/`) to the `kivy_ios/` directory. Could you please rebase/update the PR so I can merge it?
I'm on Discord if you need some support and I can also do it for you if it's too painful

**Comment by lerela** [comment]
Done, however I've not rebuilt Kivy/Numpy with the new structure to make sure it'd work!

**Comment by AndreMiras** [comment]
Thanks! And the CI built it green for you it seems :)
We make a quick check and merge

**Pull request merged**